scripts/release/test/deb/test_algorand.sh

```diff
@@ -5,6 +5,11 @@ set -ex
 export GOPATH=${HOME}/go
 export PATH=${GOPATH}/bin:/usr/local/go/bin:${PATH}
 
+PKG_NAME=algorand
+if [ "${CHANNEL}" = beta ]; then
```

Comment:
should this use double brackets?

Reply:
It's fine, I've tested it.

It uses bash builtin '[`, which should act the same as the `/usr/bin/[` binary.

Reply:
OK.